<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Feature Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_features_button">
       <property name="text">
        <string>Update Features</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="QScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollarea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>298</width>
          <height>679</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="minmax_button">
         <property name="text">
          <string>Add to Minmax</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="minmax_combobox"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="standard_button">
         <property name="text">
          <string>Add to Standard</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="standard_combobox"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="quartile_button">
         <property name="text">
          <string>Add to Quartile</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="quartile_combobox"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="noscaling_button">
         <property name="text">
          <string>Remove Scaling</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="noscaling_combobox"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="scalingsettings_button">
         <property name="text">
          <string>Show Settings</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Scaling</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="mean_button">
         <property name="text">
          <string>Add to Mean</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="mean_combobox"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="median_button">
         <property name="text">
          <string>Add to Median</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="median_combobox"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="mode_button">
         <property name="text">
          <string>Add to Mode</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="mode_combobox"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="bycategory_button">
         <property name="text">
          <string>Add to ByCategory</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="bycategory_combobox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="bycategory_category_combobox"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="byvalue_button">
         <property name="text">
          <string>Add to ByValue</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="byvalue_combobox"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="byvalue_lineedit"/>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="noimputing_button">
         <property name="text">
          <string>Remove Imputation</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="noimputing_combobox"/>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="imputesettings_button">
         <property name="text">
          <string>Show Settings</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Imputation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="polyfeatures_checkbox">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Polynomial Features</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Degree</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="polydeg_spinbox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="polyinteraction_checkbox">
           <property name="text">
            <string>Interaction Only</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Target Feature Transform</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="targettransform_combobox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="applypipeline_button">
         <property name="text">
          <string>Apply Pipeline</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Models</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Scoring</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="scoring_combobox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>CV folds</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="cvfold_spinbox">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Model Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="model_combobox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="bagging_checkbox">
       <property name="text">
        <string>Bagging (suggestion: fit base_estimator without bagging first)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="boosting_checkbox">
       <property name="text">
        <string>Boosting (suggestion: fit base_estimator without boosting first)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="paramgrid_layout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="fitmodel_button">
         <property name="text">
          <string>Fit Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="results_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="optparam_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="submission_button">
       <property name="text">
        <string>Create Submission</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plot_layout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="learningcurve_button">
         <property name="text">
          <string>Learning Curve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="validation_button">
         <property name="text">
          <string>Validation Curve</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Warnings and Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="output_lineedit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
